elastic/kibana
# [Synthetics] Disabled invalid location selector and added a badge

## Summary

Fixes https://github.com/elastic/kibana/issues/160056

### Testing

1. Create a private location against an agent policy
2. Delete an agent policy 
3. Go to add monitor view and observe location seleciton

<img width="1724" alt="image" src="https://github.com/elastic/kibana/assets/3505601/b856abee-e9b8-43f0-8de9-ae7f47e17596">
